aspnetcore/fundamentals/middleware/index.md

```diff
@@ -231,6 +231,7 @@ ASP.NET Core ships with the following middleware components. The *Order* column
 | [CORS](xref:security/cors) | Configures Cross-Origin Resource Sharing. | Before components that use CORS. |
 | [Diagnostics](xref:fundamentals/error-handling) | Configures diagnostics. | Before components that generate errors. |
 | [Forwarded Headers](/dotnet/api/microsoft.aspnetcore.builder.forwardedheadersextensions) | Forwards proxied headers onto the current request. | Before components that consume the updated fields. Examples: scheme, host, client IP, method. |
+| [Health Check](xref:host-and-deploy/health-checks) | Checks the health of ASP.NET Core infrastructure, such as database availability. | Terminal if a request matches a health check endpoint. |
```

Comment:
'Checks the health of ASP.NET Core infrastructure` makes me think that it's checking up on things about ASP.NET Core. "Checking the health of your ASP.NET Core application, such as by checking database availability" or something like that. Not sure exactly.